modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/statements/BLangIf.java

```diff
@@ -60,4 +57,24 @@ public void accept(BLangNodeVisitor visitor) {
     public NodeKind getKind() {
         return NodeKind.IF;
     }
+
+    @Override
+    public void setCondition(ExpressionNode condition) {
+        this.condition = (BLangExpression) condition;
+    }
+
+    @Override
+    public void setBody(BlockNode body) {
+        this.body = (BLangBlockStmt) body;
+    }
+
+    @Override
+    public void setElseStatement(StatementNode elseStatement) {
+        this.elseStmt = (BLangStatement) elseStatement;
+    }
+
```

Comment:
Do we really need these set methods since all fields are public? We can create ifElseStatementStack in BLangPackageBuilder.java as Stack<BLangIf> ifElseStatementStack and avoid using set methods

Reply:
Since "IfNode" is an interface, I think it's better for the interface to have the getters and setters.
We have getters and setter in all the other Node interfaces...